interfaces/Portalicious/src/app/pages/project-payments/components/create-payment/create-payment.component.ts

```diff
@@ -286,6 +286,11 @@ export class CreatePaymentComponent {
       return [];
     }
 
+    const totalPaymentAmount =
+      this.paymentAmount() * dryRunResult.sumPaymentAmountMultiplier;
+    const totalPaymentAmountFormat =
+      totalPaymentAmount % 1 === 0 ? '1.0-0' : '1.2-2';
```

Comment:
I actually think it's a bug that we're not showing decimal digits - bit of a miss from us. This should probably be the default in most places, except for wherever we show "aggregates" / big numbers (like on the payment metrics on the all payments page).

My recommendation would be:
 - use `1.2-2` always here
 - look for `1.0-0` elsewhere and check whether we are using it in a place that could lead to similar confusion

Reply:
I thought of using `1.2-2` always but I also thought 1.0-0 would look better for design reasons.
I'll change it.